### PR TITLE
Make systemd launch script agnostic of ROS version

### PR DESCRIPTION
**Pull request by Achllle**
Smol QOL update

### CODE REVIEW

File: init_scripts/launch_osr.sh [hunk 1]
```diff
@@ -5,7 +5,9 @@ set -ex
 source osr_paths.sh
 launch_dir=$OSR_CODE_DIR/ROS/osr_bringup/launch
 
-bash -c ". /opt/ros/foxy/setup.bash"
+# simple way to find the ROS 2 version. This breaks down if multiple versions of ROS are installed! In that case, hardcode your version of ROS here instead
```
Comment (abust005) [inline init_scripts/launch_osr.sh:8]: You say that it breaks down if there are multiple versions of ROS; what's the exact behavior? And should this be noted in the documentation anywhere?

**Review by abust005 — COMMENTED** [review]
Just a request for general clarification

File: init_scripts/launch_osr.sh (hunk 1 above)
Comment (apollokit) [reply init_scripts/launch_osr.sh:8]: Might be good to warn the user if it breaks down. You could do a `ls /opt/ros | wc -l` to count the number of possibilities and echo a warning if there's more than one. 

Maybe in that case you could use a user-provided argument as the ros version? Not a big deal tho.

**Review by apollokit — APPROVED** [review]
Seems fine, would be useful to warn the user if there are multiple ros versions though

File: init_scripts/launch_osr.sh (hunk 1 above)
Comment (Achllle) [reply init_scripts/launch_osr.sh:8]: I considered this but was of the opinion that having two versions of ROS installed is probably uncommon for novice builders. More experienced builders will probably be able to debug this or ask for help.

Using fancier commands has the downside of being less easy to understand/debug. That being said, I'd be willing to add this in in a follow-up PR if you guys think it's necessary!

Comment (Achllle) [reply init_scripts/launch_osr.sh:8]: For full transparency: follow-up PR by not me :smiley: )

**Pull request merged**